src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/MoveCollectionService.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service;
+
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+@Service
+public class MoveCollectionService {
```

Comment:
I'm not a fan of using a service when it's not required. What benefit comes with making a helper method a service? A price to pay is bigger spring context to load. If it comes w/o benefit it seems questionable.